pkgs/games/tworld2/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, lib
+, fetchurl
+, installShellFiles
+, qt4
+, SDL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tworld2";
+  version = "2.2.0";
+
+  src = fetchurl {
+    url = "https://tw2.bitbusters.club/downloads/tworld-${version}-src.tar.gz";
+    sha256 = "sha256:1y55v2shk2xxcds7bdwdjaq9lka31sgdp2469zqnvldchwbvcb2i";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  buildInputs = [ qt4 SDL ];
+
+  postConfigure = ''
+    echo "#define COMPILE_TIME \"$(date -ud "@$SOURCE_DATE_EPOCH" '+%Y %b %e %T %Z')\"" >comptime.h
+  '';
+
+  makeFlags = [
+    "bindir=${placeholder "out"}/bin"
+    "sharedir=${placeholder "out"}/share"
+    "mandir=${placeholder "out"}/share/man/en"
+  ];
+
+  postInstall = ''
+    cp COPYING README $out
+    mkdir $out/share/doc
+    cp docs/tworld2.html tworld.ico tworld2.ico $out/share/doc
```

Comment:
`find /nix/store -name COPYING` shows a convention in line with exactly what you instruct. Changed.

`find /nix/store -name README` shows plenty of packages that include this file (although significant inconsistencies abound). [The Debian package](https://packages.debian.org/bullseye/amd64/tworld/filelist) keeps these files. Keeping unless you insist otherwise.

`find /nix/store -name '*.ico'` shows a consensus for `$out/share/icons`. Changed.